docs/book/src/tasks/upgrading-clusters.md

```diff
@@ -0,0 +1,42 @@
+# Upgrading management and workload clusters
+
+## Considerations
+
+### Supported versions of Kubernetes
+
+If you are upgrading the version of Kubernetes for a cluster managed by Cluster API,
+check that the [running version of Cluster API on the Management Cluster supports
+the target Kubernetes version](../reference/versions.md).
+
+You may need to [upgrade the version of Cluster API](upgrading-cluster-api-versions.md)
+in order to support the target Kubernetes version.
+
+### Images
+
+For kubeadm based clusters, infrastructure providers require a "machine image"
+containing pre-installed, matching versions of `kubeadm` and `kubelet`, ensure
+that relevant infrastructure machine templates reference the appropriate image
+for the Kubernetes version.
+
+## Upgrading using the Cluster API
+
+### Order of upgrades
+
+You should always first upgrade the control plane before upgrading workload nodes, typically
+managed by `MachineDeployments`.
+
+### Upgrading the control plane
```

Comment:
there is already some documentation at https://cluster-api.sigs.k8s.io/tasks/kubeadm-control-plane.html#upgrading-the-control-plane-machines, might be good to consolidate

Reply:
I'll cross link.

Reply:
Consolidated into this document, and cross-referenced from the kcp doc.